### PR TITLE
Prevent the page to scroll when modal search/menu is open [Fixes #4194]

### LINKED ISSUE #4194: Background is scrollable when navigation modal is open on smaller screen sizes
**Describe the bug**
On screens below 1025px wide the background is scrollable when the navigation modal is open.

[View a GIF of the bug](https://i.gyazo.com/650bdaffedef83761429237b7b08386a.gif)

**To Reproduce**

1. Go to [ethereum.org](https://ethereum.org/en/)
2. Reduce screen size below 1025px wide
3. Click on navigation hamburger
4. See scrolling error

**Expected behavior**

Scrolling should be limited to the sidebar modal.

**Additional context**
We are dealing with a similar issue in https://github.com/ethereum/ethereum-org-website/pull/3799

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

### PR DESCRIPTION
**Pull request by JulienMaurice**
## Description

Prevent the page to scroll when the screen is below 1025px wide and the navigation modal are open

Done by just stopping gatsby to force `overflow-y: scroll` when the hamburger menu or the search bar are open.

Done with @vluna
## Related Issue

 #https://github.com/ethereum/ethereum-org-website/issues/4194#event-5581859174

### CODE REVIEW

**Review by minimalsm — APPROVED** [review]
Looks like an okay solution to me. Just waiting on another review ☺️

**Review by corwintines — APPROVED** [review]
LGTM! Thanks @JulienMaurice!

**Comment by corwintines** [comment]
@all-contributors can you add @JulienMaurice for code

**Pull request merged**
